### ISSUE TITLE
Failed to establish a new connection

### ISSUE THREAD

**Issue by shuuchen** [gbeced/pyalgotrade#101]
Hi All,
I tried the tutorial from http://gbeced.github.io/pyalgotrade/docs/v0.18/html/tutorial.html
where I ran the code 
```
python -c "from pyalgotrade.tools import yahoofinance; yahoofinance.download_daily_bars('orcl', 2000, 'orcl-2000.csv')"
```

then I met with such error messages

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/anaconda/lib/python2.7/site-packages/pyalgotrade/tools/yahoofinance.py", line 54, in download_daily_bars
    bars = download_csv(instrument, datetime.date(year, 1, 1), datetime.date(year, 12, 31), "d")
  File "/anaconda/lib/python2.7/site-packages/pyalgotrade/tools/yahoofinance.py", line 40, in download_csv
    return csvutils.download_csv(url)
  File "/anaconda/lib/python2.7/site-packages/pyalgotrade/utils/csvutils.py", line 57, in download_csv
    response = requests.get(url, params=url_params)
  File "/anaconda/lib/python2.7/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/anaconda/lib/python2.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 518, in request
    resp = self.send(prep, **send_kwargs)
  File "/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 639, in send
    r = adapter.send(request, **kwargs)
  File "/anaconda/lib/python2.7/site-packages/requests/adapters.py", line 502, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='ichart.finance.yahoo.com', port=80): Max retries exceeded with url: /table.csv?s=orcl&a=0&b=1&c=2000&d=11&e=31&f=2000&g=d&ignore=.csv (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x112983390>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
```

Does anyone have the same issue?

**Comment by blakejennings** [comment]
Ya - Yahoo Finance iCharts are broken. See issue #98.

**Comment by shuuchen** [comment]
OK.Thanks!

**Comment by shuuchen** [comment]
So any tutorials for new learners?

**Comment by gbeced** [comment]
Keep this closed since its a duplicate of #98. I'm working on an updated version that uses Quandl for the tutorial and some examples.
